src/qt/res/css/trad.css

```diff
@@ -25,6 +25,43 @@ Loaded in GUIUtil::loadStyleSheet() in guitil.cpp.
 </colors>
 */
 
+/******************************************************
+QToolBar / QToolButton
+******************************************************/
+
+QToolBar {
+    border: none;
+    width: 100%;
+    padding: 0;
+    margin: 0;
+    spacing: 0;
+}
+
+QToolBar > QToolButton {
+    background-color: #008de4;
+    border: none;
+    min-height: 50px;
+    font-size: 16px;
+    color: #fff;
+    width: 100%;
+    padding: 1px;
+    text-align: center;
+}
+
+QToolBar > QToolButton:hover:!checked {
+    background-color: #005e98;
+    color: #DCDCDC;
+}
+
+QToolBar > QToolButton:checked {
+    background-color: #00000000;
+    color: #333;
+}
+
+QToolBar > QToolButton:disabled {
+    color: #444;
+}
+
```

Comment:
I'd keep styling for the Traditional scheme minimal i.e.
```suggestion

```

<img width="961" alt="Screenshot 2020-06-27 at 14 16 43" src="https://user-images.githubusercontent.com/1935069/85920993-40762d00-b881-11ea-93ba-f271241b82cd.png">

Reply:
I agree with keeping traditional limited